tools/cpp/windows_cc_configure.bzl

```diff
@@ -296,9 +326,8 @@ def configure_windows_toolchain(repository_ctx):
   repository_ctx.symlink(Label("@bazel_tools//tools/cpp:BUILD.static"), "BUILD")
 
   vc_path = find_vc_path(repository_ctx)
```

Comment:
Could you change `find_vc_path` to return `None` when the path is not found? See my other comment in `_check_vc_installation` why I prefer that.